src/main/java/net/foxes4life/foxclient/gui/BlockHud.java

```diff
@@ -0,0 +1,120 @@
+package net.foxes4life.foxclient.gui;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.foxes4life.foxclient.Main;
+import net.foxes4life.foxclient.configuration.FoxClientSetting;
+import net.foxes4life.foxclient.util.TextUtils;
+import net.foxes4life.foxclient.util.draw.Anchor;
+import net.foxes4life.foxclient.util.draw.AnchoredBounds;
+import net.foxes4life.foxclient.util.draw.DrawUtils;
+import net.foxes4life.foxclient.util.rendering.ItemRender;
+import net.minecraft.block.AirBlock;
+import net.minecraft.block.Block;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.DrawContext;
+import net.minecraft.client.gui.hud.InGameHud;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.client.network.ClientPlayerInteractionManager;
+import net.minecraft.client.render.RenderLayer;
+import net.minecraft.client.render.item.ItemRenderer;
+import net.minecraft.client.render.model.json.ModelTransformationMode;
+import net.minecraft.client.world.ClientWorld;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.text.MutableText;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.RaycastContext;
+
+import java.awt.*;
+
+public class BlockHud {
+    private final MinecraftClient client;
+    private Block lastBlock;
+
+    private float y = 0;
+    private float width = 0;
+
+    public BlockHud(MinecraftClient client) {
+        this.client = client;
+    }
+
+    public void render(DrawContext context, ItemRender itemRender) {
+        Block block = getBlock();
+        boolean show = false;
+
+        if (block != null && !(block instanceof AirBlock)) {
+            lastBlock = block;
+            show = true;
+        }
+        else {
```

Comment:
style: please put `else` on the same line as the `}` belonging to the original if statement